### PR TITLE
glibmm

### PR DESCRIPTION
**Pull request by scopatz**
Not sure if this is going to work with the state of gcc.

### CODE REVIEW

**Comment by scopatz** [comment]
Uggh and I forgot to skip appveyor. Sorry!

**Comment by jakirkham** [comment]
> Not sure if this is going to work with the state of gcc.

So, I have done my best to fix it this morning. We have tested it in a few cases and found it works.

However, it is a duct tape job. The state of the `gcc` package is subject to breakage if any of its dependencies change again (not impossible as some are shared with things like SymPy). Plus, we can't seem to rebuild it ourselves (CI time limits) nor will it be rebuilt by Continuum in the near term. Even if it is rebuilt somehow, it is not clear to me that something else will break it (e.g. `gmp` and `mpfr` have lower version bounds but no upper version bounds).

I'm trying to push us away from using the `gcc` package at all on Linux and use [devtoolset-2.1]( http://linux.web.cern.ch/linux/devtoolset/#dts21 ) (was folded into 2 basically), which supports `gcc` 4.8.2 (we use 4.8.5 now) in the near term ( https://github.com/pelson/Obvious-CI/pull/61 ). I don't think this will be the last time we switch compilers as we may want more features (e.g. C++14 support). However, this is similar to what [manylinux]( https://github.com/pypa/manylinux ) does just on CentOS 6 (our current OS). I have gotten at least some support. Hoping to get more people to buy in so we can leave the `gcc` package behind.

There are still a couple of use cases for `gcc` on Mac, but that ends up being if Fortran or OpenMP is used. All other cases like C++0x and C++11 support, which are far more frequent, are already handled well by the system compiler.

**Comment by scopatz** [comment]
This finally works! A review / merge would be most welcome. Thanks!

File: recipes/glibmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - libsigcpp
+    - glib
+    - xz
```
Comment (ocefpaf) [inline recipes/glibmm/meta.yaml]: We are [recommending](https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies) to pin libzma as `xz 5.0.*`. That way we avoid getting a modern version and being incompatible with Python 3.x

**Comment by ocefpaf** [comment]
LGTM @scopatz. Just a minor comment that you may not think it matters here.

File: recipes/glibmm/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipes/glibmm/meta.yaml]: Nice catch! Sorry about that.

**Comment by scopatz** [comment]
Thanks @ocefpaf! Looks like this is ready to go now.

**Pull request merged**